modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,218 @@
+= Recovery Mode
+:description: Recovery mode starts Redpanda in "metadata only" mode and disables partitions so you can repair a failed cluster.
+
+Recovery mode allows you to start Redpanda and form a stable cluster when it fails due to a system crash, OOM errors, or other issues that make a normal startup infeasible. 
+
+In recovery mode, cluster configuration changes and other manual administrative actions are permitted so you can repair and restore the failed cluster. Topic reads and writes are disabled.
+
+== About recovery mode
+
+Recovery mode starts a cluster with limited functionality and deactivates normal traffic and data operations for a cluster. Redpanda does not load user-managed partitions on disk. This helps prevent triggering a storm of partition leader elections and replication (such as increased learner replica recovery load after a long broker downtime) that may occur on startup. 
+
+Normal traffic and especially malicious traffic can disrupt availability for all users, including admin users. Recovery mode helps avoid placing the cluster under a heavy load of Kafka API and HTTP Proxy connections as you work to resuscitate the cluster.
```

Comment:
@JakeSCahill Yes exactly. Hm, this is how it was written in the PRD:

> Recovery mode also intends to avoid the load of Kafka API/HTTP Proxy connections, because normal traffic and especially abusive/rogue traffic can prevent availability for all users, even the admin trying to resuscitate the cluster.

I took it to mean that Kafka API and HTTP Proxy in particular can see heavy enough usage to be disruptive when trying to troubleshoot/fix a cluster. I'm not sure why SR isn't specifically mentioned there, but yes it is not available in recovery mode either.

Reply:
>  all APIs are disabled apart from the Admin API?

Kafka API is only partially disabled (fetch and produce requests return non-recoverable errors). The reason for this is that some administrative actions (like deleting topics) that are needed in recovery mode are only available via Kafka API.

Reply:
Thanks @ztlpn - @kbatuigas we should clarify this in the list of enabled and disabled actions

Reply:
@JakeSCahill @ztlpn added to "disabled" list. "Delete topics" is already in the enabled list -- should "Kafka API" be specified with it as well?

Reply:
Looks great. I would flip the two so that users can see what's enabled first.

Reply:
It would be nice to specify which actions are enabled for the Kafka API

Reply:
Right, we should state that certain Kafka API is partially disabled and certain administrative requests are still available (deleting topics, modifying topic properties, editing consumer group metadata) - this is from the list of enabled operations, maybe grouping them by API type is a good idea.